torchdynamo/side_effects.py

```diff
@@ -300,7 +310,12 @@ def codegen(self, cg: PyCodegen):
                     cg.tx.output.update_co_names(name)
                     cg(value)
                     cg(var.mutable_local.source)
-                    suffixes.append([create_instruction("STORE_ATTR", name)])
+                    op = (
+                        "STORE_GLOBAL"
```

Comment:
Doesn't `STORE_GLOBAL` pop 1 from the stack (while STORE_ATTR pops 2).  Are we leaking a value onto the stack here?